cirq/ops/named_qubit_test.py

```diff
@@ -18,9 +18,10 @@
 
 def test_named_qubit_str():
     q = cirq.NamedQubit('a')
-    assert q.name == 'a'
+    assert q._name == 'a'
```

Comment:
This is an incorrect change. The name property should still be exposed. Add a `@property def name` getter for it.